### PR TITLE
Add robots.txt and fix image bug

### PR DESCRIPTION
**Pull request by johanarybarova**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -10,6 +10,9 @@ export const metadata = {
 
 const RootLayout: ChildrenFC = ({ children }) => (
   <html lang="cs">
+    <head>
+      <meta name="robots" content="noindex, nofollow" />
```
Comment (GLObus303) [inline src/app/layout.tsx:14]: Is there no need to change the name to the path with an extension? Is this enough? Something like `./robots.txt` or just `robots.txt`. 

I am not familiar with it, so teach me

File: src/components/Events/EventCard.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const EventCard: React.FC<EventCardProps> = ({
     setIsLiked(!isLiked);
   };
 
-  const getImagePath = (imageId: number) => `/${(imageId % 12) + 1}.png`;
+  const getImagePath = (eventPrice: number) => `/${(eventPrice % 12) + 1}.png`;
```
Comment (GLObus303) [inline src/components/Events/EventCard.tsx:28]: Is the image based on price? I would expect sport, id or something 😂

**Review by GLObus303 — APPROVED** [review]

File: src/components/Events/EventCard.tsx (hunk 2 above)
Comment (johanarybarova) [reply src/components/Events/EventCard.tsx:28]: This is for now just randomized images - I will have a PR with a folder/images for each sport, but currently, I don't have a list of available sports. I need the list from either the backend, or we need to create a list of, for example, 10 sports for now and scale it later. Before the bug, it was still random based on the id number. Before, it worked with the ID, but later you changed it to something like vx5hu4578sc67... soooo 😅

File: src/app/layout.tsx (hunk 1 above)
Comment (johanarybarova) [reply src/app/layout.tsx:14]: this shoud be correct - for next.js it is robots.txt inside /public directory  👇
https://nextjs.org/learn-pages-router/seo/crawling-and-indexing/metatags
https://nextjs.org/learn-pages-router/seo/crawling-and-indexing/robots-txt

File: src/components/Events/EventCard.tsx (hunk 2 above)
Comment (GLObus303) [reply src/components/Events/EventCard.tsx:28]: :D . Stupid backend people, right?

**Pull request merged**